test/TellorPriceOracleWETH.t.sol

```diff
@@ -4,17 +4,17 @@ pragma solidity 0.8.19;
 import { Test } from "forge-std/Test.sol";
 import { ITellor } from "../contracts/Dependencies/ITellor.sol";
 import { IPriceOracle } from "../contracts/Oracles/Interfaces/IPriceOracle.sol";
-import { TellorPriceOracleWETH } from "../contracts/Oracles/TellorPriceOracleWETH.sol";
+import { TellorPriceOracle } from "../contracts/Oracles/TellorPriceOracle.sol";
 
 contract TellorPriceOracleWETHTest is Test {
     ITellor public constant tellorOracle = ITellor(0xD9157453E2668B2fc45b7A803D3FEF3642430cC0);
-    TellorPriceOracleWETH public tellorPriceOracleWETH;
+    TellorPriceOracle public tellorPriceOracleWETH;
 
     function setUp() public {
         vm.createSelectFork("mainnet", 17_484_688);
 
         tellorPriceOracleWETH =
-            new TellorPriceOracleWETH(tellorOracle, keccak256(abi.encode("SpotPrice", abi.encode("eth", "usd"))));
+            new TellorPriceOracle(tellorOracle, keccak256(abi.encode("SpotPrice", abi.encode("eth", "usd"))), 5e15);
```

Comment:
```suggestion
            new TellorPriceOracle(tellorOracle, keccak256(abi.encode("SpotPrice", abi.encode("eth", "usd"))), 0);
```

Reply:
currently the Tellor deviation constant is set to 5e15 though

Reply:
Yeah, in reality, it is not a problem because it is still less than Chainlink, but, in real, it is 0.